.github/workflows/check-move-packages-pull-request.yml

```diff
@@ -40,11 +38,6 @@ jobs:
       - name: Run move-vm-runtime tests
         run: cargo test -p move-vm-runtime
 
-      # move-compiler
-
-      - name: Run move-compiler tests
-        run: cargo test -p move-compiler
-
```

Comment:
This lasts for 3.5 minutes, and we will run this on push action, so I removed it from here.